### PR TITLE
WIP: Feature/increase dashboard tests coverage

### PR DESCRIPTION
**Pull request by cryptomental**
WIP: This PR is opened to force CI to provide coverage numbers until satisfied.

### CODE REVIEW

**Comment by cryptomental** [comment]
Haha omg 0.29% more coverage after yesterday's session. I strive for perfection and could achieve > 95% total coverage but this bounty would take me a few week. I need to think of a smarter approach i.e. add tests that will cover more lines of code.

**Comment by cryptomental** [comment]
I thought about my previous comment for a while and came up to the conclusion that blindly increasing test coverage just for the number won't be that smart and perhaps it is better to target an area that requires attention. I had a look at dashboard/models.py 

https://codecov.io/gh/gitcoinco/web/src/dd99429962eb1ae7252ffb4edb25fa37cbb412a0/app/dashboard/models.py

Providing 100% coverage on this file on top of the tests that I already added will require a few hours of work and might be a sensible solution for this bounty. What is your view on this @owocki @mbeacom ?

**Comment by owocki** [comment]
i think that dashboard/models would be a great place to focus

**Comment by cryptomental** [comment]
Excellent,I will try to reach 100% in dashboard/models. I will continue on this tomorrow.

**Comment by cryptomental** [comment]
@owocki @mbeacom I hereby confirm the 4% goal was achieved.

https://www.youtube.com/watch?v=iwGFalTRHDA

**Comment by cryptomental** [comment]
I tried to 'Submit Work' but transaction failed likely to expired bounty https://etherscan.io/tx/0x8529fe66b2867cc413b860c79343e4347e30bb66fb77f7126fe07c5b30cfe2a1 we can open a new one or simply close that one and tip 0.12 eth as agreed.

File: app/dashboard/tests/test_dashboard_embed.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          accepted=False,
+          network='mainnet',
+          expires_date=datetime(2222, 11, 30, tzinfo=pytz.UTC),
+          idx_project_length=5,
+          project_length='Months',
+          bounty_type='Feature',
+          experience_level='Intermediate',
+          raw_data={},
+        )
+        assert embed(self.factory.get('https://github.com/gitcoinco/web/issues/11?repo=https://github.com/gitcoinco/web')).status_code in [200, 422]
```
Comment (owocki) [inline app/dashboard/tests/test_dashboard_embed.py]: this doesnt really test anything meaningful about the logic (other than that it generates an image i suppose)

**Comment by owocki** [comment]
just left a comment or two.. @mbeacom what do you think

**Comment by cryptomental** [comment]
I am very positive about this one, 20+ test cases and just one comment. @mbeacom does it look OK for you to merge?

**Comment by owocki** [comment]
ok to merge this? looks like travis failed

**Comment by mbeacom** [comment]
@cryptomental After rebasing this branch, I'm still seeing a failure in:  ` - app/dashboard/tests/test_notifications.py:82 DashboardNotificationsTest.test_maybe_post_on_craigslist`

**Comment by mbeacom** [comment]
@cryptomental Do you want me to update your tests to pass Travis or would you like to do it?

Thanks

**Comment by mbeacom** [comment]
@cryptomental Do you still want to have this merged? If so, can you resolve the broken tests and resolve the conflict?  If not, feel free to let us know and we'll set aside some time to rebranch this and get it merged.

Thanks!

**Pull request merged**